extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -384,6 +387,8 @@ public function getArticle() {
 			if ( !empty( $relatedPages ) ) {
 				$data['relatedPages'] = $relatedPages;
 			}
+
+			$data['htmlTitle'] = $this->wg->Out->getWikiaHTMLTitle( $title->getPrefixedText(), $title->isMainPage() );
```

Comment:
we have the logic to get it in MercuryApi.class.php
\MercuryApi::getSiteMessage

is it possible to reuse it?